kitsune/settings.py

```diff
@@ -45,13 +49,13 @@ def path(*parts):
         # Or path to database file if sqlite3.
         'NAME': 'kitsune',
         # Not used with sqlite3.
-        'USER': '',
+        'USER': config('DATABASE_USER'),
```

Comment:
I have done this in the commit https://github.com/mozilla/kitsune/pull/2884/commits/a6c529a222e47f41b216d53f81b5df1fa84aa299
But it was failing some of the tests. so needed to revert it back!